ru/lessons/advanced/otp-supervisors.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: OTP Супервизоры
+category: advanced
+order: 6
+lang: ru
+---
+
+Супервизоры это специальные процессы с единственной целью: мониторинг других процессов. Супервизоры позволяют нам создавать отказоустойчивые приложения при помощи автоматического перезапуска дочерних процессов в случае их отказа.
+
+## Содержание
+
+- [Настройка](#section-1)
+  - [Стратегии](#section-2)
+  - [Вложенность](#section-3)
+- [Наблюдатель задач](#section-4)
+  - [Настройка](#section-5)
+  - [Наблюдаемые задачи](#section-6)
+
+## Настройка
+
+Магия Супервизоров заключена в функции `Supervisor.start_link/2`.  Помимо запуска нашего Супервизора и его потомков, она позволяет нам определять стратегию, которую будет использовать наш Супервизор для управления дочерними процессами.
```

Comment:
Why do we need to write 'supervisor' from the capital letter in this lesson? 

If it's mentioning class - then we need to use english term Supervisor.
If it's mentioning concept - then we can write it from the downcased letter.

What do you think about this?